pkg/api/kubegraph/edges.go

```diff
@@ -148,14 +153,18 @@ func AddMountedSecretEdges(g osgraph.Graph, podSpec *kubegraph.PodSpecNode) {
 		syntheticSecret.Name = source.Secret.SecretName
 
 		secretNode := kubegraph.FindOrCreateSyntheticSecretNode(g, syntheticSecret)
+		kind := fmt.Sprintf("%vs", strings.ToLower(secretNode.Kind()))
+		if forbiddenResources.Has(kind) {
+			g.AddEdge(podSpec, secretNode, ForbiddenResourceEdgeKind)
```

Comment:
@deads2k 
> wasn't expecting an edge. Can you use the failure list to inform which markers you place?

I'm not sure how I would "pipe" the list of forbidden resources to where I set the markers in [kubegraph/analysis/podspec.go](https://github.com/openshift/origin/blob/master/pkg/api/kubegraph/analysis/podspec.go#L70). [These](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/projectstatus.go#L365) are the only markers that seem to be placed with regards to forbidden resources in [cmd/cli/describe/projectstatus.go](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/projectstatus.go#L365)